### ISSUE TITLE
Terminal not recognizing ESLint as a command

### ISSUE THREAD

**Issue by organism** [eslint/eslint#10494]
2017 MacBook 12", macOS Sierra 10.12.6

* **ESLint Version:** 4.19.1
* **Node Version:** 10.4.1
* **npm Version:** 6.1.0

**What parser (default, Babel-ESLint, etc.) are you using?**

Default

**Please show your full configuration:**

No config. Issues at install.

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

No source.

```bash
eslint --init
```

**What did you expect to happen?**
ESLint should have made a local configuration.

**What actually happened? Please include the actual, raw output from ESLint.**
```bash
-bash: eslint: command not found
```

This is the weird part. ESLint shows that it's fully installed and up and running:

```
[Info  - 12:27:03] ESLint server is running.
[Info  - 12:27:04] ESLint library loaded from: /fakepath/node_modules/eslint/lib/api.js
[Warn  - 12:27:04] 
No ESLint configuration (e.g .eslintrc) found for file: /fakepath/index.js
File will not be validated. Consider running 'eslint --init' in the workspace folder Workspace
Alternatively you can disable ESLint by executing the 'Disable ESLint' command.

```

**Comment by platinumazure** [comment]
What installation steps did you do to create ESLint?

There is no ESLint server in ESLint core-- is this being run from an editor integration or some other wrapper?

**Comment by organism** [comment]
@platinumazure It's running from VSCode w/ the extension.

I used `npm install --save eslint`, and also tried installing it globally, but it still is not recognized.

**Comment by platinumazure** [comment]
Hmm. `npm install --save eslint` would have made the binary available only under `./node_modules/.bin/eslint` (relative to your project root), so `eslint` shouldn't have worked. But a global install should have made it work.

Can you run ESLint on the command line? If so, the problem is with your VSCode configuration...

**Comment by organism** [comment]
It does not work in the command line.

**Comment by jukebox017** [comment]
Does runing `node_modules/.bin/eslint --init` work?

**Comment by organism** [comment]
That does work from home, yes. @jukebox017

**Comment by jukebox017** [comment]
You can run commands in `node_modules` using `npx`:

```
npx eslint .
```

https://stackoverflow.com/a/45164863

**Comment by nzakas** [comment]
The last comment answered the question, so I'm closing the issue. If you still need help, please send a message to our [mailing list](https://groups.google.com/group/eslint) or [chatroom](https://gitter.im/eslint/eslint). Thanks!
